builtin/game/chatcommands.lua

```diff
@@ -84,6 +84,15 @@ core.register_chatcommand("me", {
 	end,
 })
 
+core.register_chatcommand("admin", {
+	params = "<action>",
+    privs = {shout = true},
+    description = "Show the name of the server owner",
+    func = function(name)
+        return true, (minetest.setting_get("name") or "Couldn't determine who") .. " is the owner of this server"
```

Comment:
This should return false if it can't find an owner name, also the line is too long.
Try this:
```
local admin = minetest.setting_get("name")
if admin then
	return true, "The administrator of this server is "..admin.."."
else
	return false, "There's no administrator named in the config file."
end